dapps/assets/src/components/contacts/Contacts.ts

```diff
@@ -149,4 +174,11 @@ export default class ContactsComponent extends mixins(EvanComponent) {
     this.filterBy = [];
     this.filter = null;
   }
+
+  onSearchChange = debounce(this.filterBySearchTerm, 250);
```

Comment:
shall we make the debounce in the search-box component and emit a searchChanged event?

I think there would be no use case, where not debouncing is required.